pkg/services/alerting/notifiers/hipchat.go

```diff
@@ -37,6 +37,15 @@ func init() {
           data-placement="right">
         </input>
       </div>
+			<div class="gf-form max-width-30">
+        <gf-form-switch
+           class="gf-form max-width-30"
+           label="Plain Text"
+           label-class="width-8"
+           checked="ctrl.model.settings.plainText"
+           tooltip="Send Hip Chat messages as text instead of the default card (Allows for more detailed messages).">
```

Comment:
Hey @torkelo,

I think the reason for both is to allow backwards capability, Hipchat cards are an optional override to the traditional message... Cards are only displayed in supported clients and the message field is used in clients that do not support cards.

Cards allow a title, image and short description, as an observation the short description is truncated and therefore doesn't display the useful information in the alert, however, I haven't found any supporting information that suggests what the limit is.

Sam